frontend/src/components/search/PublicSearchResult.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+import {
+	Box,
+	Link as MuiLink,
+	List,
+	ListItem,
+	ListItemAvatar,
+	Typography,
+} from "@mui/material";
+import { Link } from "react-router-dom";
+import DatasetIcon from "@mui/icons-material/Dataset";
+import InsertDriveFileIcon from "@mui/icons-material/InsertDriveFile";
+import { parseDate } from "../../utils/common";
+import { theme } from "../../theme";
+
+import parse from "html-react-parser";
+
+// Function to parse the elastic search parameter
+// If it contains HTML tags like <mark>, it removes them
+// If there is no HTML tags, it returns the original string
+function parseString(str: string) {
+	try {
+		const parsedHtml = parse(str);
+		return parsedHtml;
+	} catch (error) {
+		return str;
+	}
+}
+
+function buildDatasetResult(item) {
+	return (
+		<>
+			<ListItemAvatar sx={{ color: theme.palette.primary.main }}>
+				<DatasetIcon />
+			</ListItemAvatar>
+			<Box sx={{ marginTop: "5px" }}>
+				<MuiLink
+					component={Link}
+					to={`/public_datasets/${item._id}`}
+					sx={{ fontWeight: "bold", fontSize: "18px" }}
```

Comment:
here too...looks like you're using the same styling properties again.

Reply:
I created a separate issue for these.